modoboa/admin/management/commands/subcommands/_mx.py

```diff
@@ -0,0 +1,201 @@
+"""Management command to check defined domains."""
+
+import ipaddress
```

Comment:
Could you move this import after datetime to respect the alphabetical order please ?